src/github.com/getlantern/flashlight/settings/settings.go

```diff
@@ -92,14 +91,6 @@ func read() {
 		err := config.Update(func(updated *config.Config) error {
 
 			if autoReport, ok := settings["autoReport"].(bool); ok {
-				// turn on/off analaytics reporting
-				if autoReport {
-					if err := analytics.StartService(); err != nil {
-						log.Debugf("Unable to start analytics service: %v", err)
-					}
-				} else {
-					analytics.StopService()
-				}
```

Comment:
There is the case where if the user enables auto-reporting, after having it disabled, we don't immediately track the session until Lantern restarts, but since auto-reporting is the default now that shouldn't be much of a problem anyway.